README.md

```diff
@@ -18,15 +18,31 @@ cp .env.example .env
 
 Required env variables to run:
 
-- `FUNDER_ACCOUNT_PRIVATE_KEY`
-- `GELATO_RELAY_API_KEY`
-- `RPC_URL`
-- `CHAIN_ID`
+```
+FUNDER_ACCOUNT_PRIVATE_KEY=
+GELATO_RELAY_API_KEY=
+RPC_URL=
+CHAIN_ID=
+```
+
+In some networks, as development, you must specify the deployed Safe contract addresses with the following env vars:
+
+```
+SAFE_MASTER_COPY_ADDRESS=
+SAFE_PROXY_FACTORY_ADDRESS=
+FALLBACK_HANDLER_ADDRESS=
+MULTI_SENT_ADDRESS=
+MULTI_SEND_CALL_ONLY_ADDRESS=
+```
 
 ### Run
 
 ```bash
-$ npm start
+npm start
```

Comment:
In the README it's also missing the documentation for tests. I mean including the `npm test`